src/theme.rs

```diff
@@ -1,55 +1,95 @@
 use termcolor::{Color, ColorSpec};
 
 /// Theme for styling the UI.
+///
+/// # Example
+///
+/// ```
+/// let custom_theme = Theme {
```

Comment:
```suggestion
/// use demand::Theme;
/// let custom_theme = Theme {
```

this makes `cargo test` work since rust will actually try to compile this and run it

Reply:
Good hint, thx! My IDE even shows me an action to run this...

Reply:
yeah though I don't know how useful it is for this project since I'm not sure if it makes sense to run a lot of the tests that create input. It _is_ nice that it at least compiles this code though.

Reply:
I feel like I'd like it if `cargo test` was able to at least execute, right now it hangs, presumably waiting for input for one of the examples that doesn't even display.

Reply:
Yes it hangs for all the inputs since `run()` is blocking. I also wonder how we can unit test this, would be nice to have some unit tests for the inputs. Will try to figure out if we can do something here...

Reply:
I think executing the demos in CI might be an easier way to have automated tests